router/router.py

```diff
@@ -316,7 +316,7 @@ def sort_compiled(self):
                 paths = list(self.compiled[pathkind][qkind].keys())
                 paths.sort()
                 for path in paths:
-                    # see `compile_resulte docs for line_modifier above.
+                    # see `compile_result` docs for line_modifier above.
```

Comment:
this is a little unusual

Reply:
I am very proficient at making interesting typos! :)